apps/spruce/src/pages/version/TestAnalysis/GroupedTestMapList/FailedTestGroup/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import TaskStatusBadge from "@evg-ui/lib/components/Badge/TaskStatusBadge";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { TaskStatus } from "@evg-ui/lib/types/task";
+import { Accordion } from "components/Accordion";
+import { trimStringFromMiddle } from "utils/string";
+import { TaskBuildVariantField } from "../../types";
+import FailedTestGroupTable from "./FailedTestGroupTable";
+
+interface FailedTestGroupProps {
+  testName: string;
+  tasks: TaskBuildVariantField[];
+}
+
+const FailedTestGroup: React.FC<FailedTestGroupProps> = ({
+  tasks,
+  testName,
+}) => {
+  const statusMap = countStatuses(tasks);
+  const sortedStatuses = Object.entries(statusMap).sort((a, b) =>
+    a[0].localeCompare(b[0]),
+  );
+  return (
+    <Accordion
+      caretAlignSelf="baseline"
+      shouldRenderChildIfHidden={false}
+      title={
+        <TitleContainer>
+          <Title allowWrap={false} title={testName}>
```

Comment:
I think the approach before was better (I don't really like this `allowWrap` pattern). Let's just leave it for now because I can't figure out the best way to display this. I suspect something is wrong with the design pattern in the first place and that accordion titles should never be that long